Apps/Inc/SendTritium.h

```diff
@@ -12,7 +12,7 @@
 
 #include "common.h"
 
-#define SENDTRITIUM_PRINT_MES
+//#define SENDTRITIUM_PRINT_MES
```

Comment:
just commenting this out is kinda inelegant imo. Could we make this a makefile flag maybe? That way we don't have to change code when we want this, we could just build a different binary.

Reply:
Just created #384 to address this. Not important for MVP, will be addressed in M2.0.1